misc/services/mobile-webui/mobile-webui-frontend-testing/tests/spec/distribution.spec.js

```diff
@@ -45,12 +47,13 @@ const createMasterdata = async () => {
         dropToLocatorQRCode: response.warehouses.wh2.locatorQRCode,
         launcherTestId: response.distributionOrders.DD1.launcherTestId,
         huQRCode: response.handlingUnits.HU1.qrCode,
+        lineQty: lineInfo.qtyEntered,
     };
 }
 
 // noinspection JSUnusedLocalSymbols
 test('Simple distribution test', async ({ page }) => {
-    const { login, warehouseFromFacetId, launcherTestId, huQRCode, dropToLocatorQRCode } = await createMasterdata();
+    const { login, warehouseFromFacetId, launcherTestId, huQRCode, lineQty, dropToLocatorQRCode } = await createMasterdata();
```

Comment:
tbh, i would prefer to call createMasterdata({ qtyToMove: 100}) instead of returning `lineQty`.

Also below, i would hardcode the values.

reason: the test shall define the context and not vice-versa. 
createMasterdata() returns only the values that are needed to run the test, but are not relevant for the test / polutes the test readabilty.